ipywidgets/static/widgets/js/widget_bool.js

```diff
@@ -10,8 +10,24 @@ define([
     "bootstrap",
 ], function(widget, $){
 
+    var BoolModel = widget.DOMWidgetModel.extend({
+        defaults: _.extend(widget.DOMWidgetModel.prototype.defaults, {
+            value: false,
+            description: "",
+            disabled: false,
+            _model_name: "BoolModel"
+        }),
+    });
+
+    var CheckboxModel = BoolModel.extend({
+        defaults: _.extend(BoolModel.prototype.defaults, {
```

Comment:
This is the line that is causing the tests to fail.  You need to import underscore.